derrida/books/models.py

```diff
@@ -638,37 +393,11 @@ class DerridaWork(Notable):
     full_citation = models.TextField()
     #: boolean indicator for primary work
     is_primary = models.BooleanField()
-    cited_books = models.ManyToManyField(
-        'Book',
-        through='DerridaWorkBook',
-        help_text='Indicate which edition(s) of a book Derrida '
-                  'cites in this work.',
-    )
 
     def __str__(self):
         return self.short_title
 
 
-class DerridaWorkBook(Notable):
```

Comment:
This is no longer needed because Instance model has a `cited_in` many-to-many relationship to DerridaWork model.